Makefile

```diff
@@ -1,7 +1,22 @@
-all:
-	tree-sitter generate
-	tree-sitter test
+all: test
+
+build:
+	npm run build
+
+test: build
+	npm run test
 
 run: all
-	tree-sitter build-wasm
-	tree-sitter web-ui
+	npm run wasm
+	npm run web
```

Comment:
replaced all these, so they are run using the tree-sitter binary from the project, and not the global one.